src/services/extensions/exposedComponents.tsx

```diff
@@ -19,11 +21,10 @@ function SuspendedOpenInLogsDrilldownButton(props: OpenInLogsDrilldownButtonProp
   );
 }
 
-export const exposedComponents = [
-  {
-    component: SuspendedOpenInLogsDrilldownButton,
-    description: 'A button that opens a logs view in the Logs Drilldown app.',
-    id: `grafana-lokiexplore-app/open-in-explore-logs-button/v1`,
-    title: 'Open in Logs Drilldown button',
-  },
-];
+export function SuspendedEmbeddedLogsExploration(props: EmbeddedLogsExplorationProps) {
+  return (
+    <Suspense fallback={<div>Loading...</div>}>
```

Comment:
```suggestion
    <Suspense fallback={<div>Loading Logs application</div>}>
```

IDk. Sounds a bit less dry.